docs/csharp/how-to/index.md

```diff
@@ -0,0 +1,241 @@
+---
+title: How to articles (C# Guide)
+description: A collection of quick tips and short, focused code samples
+author: billwagner
+ms.author: wiwagn
+ms.date: 12/20/2017
+ms.topic: article
+ms.prod: .net
+ms.devlang: devlang-csharp
+---
+
+# How to (C#)
+
+The How to section of the C# Guide is where you can find quick answers
+to common questions. There are cases on this page were one article may
+be listed in multiple sections. We wanted to make them easy to find
+for multiple search paths. 
+
+## General C# concepts
+
+There are several tips and tricks that are common C# developer practices.
+
+- [Initialize objects using an object initializer](../programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer.md).
+- [Learn the differences between passing a struct and a class to a method](../programming-guide/classes-and-structs/how-to-know-the-difference-passing-a-struct-and-passing-a-class-to-a-method.md).
+- [How to use lambda expressions](../programming-guide/statements-expressions-operators/how-to-use-lambda-expressions-outside-linq.md).
+- [Resolve type name conflicts by using the global namespace alias](../programming-guide/namespaces/how-to-use-the-global-namespace-alias.md).
+- [Use operator overloading](../programming-guide/statements-expressions-operators/how-to-use-operator-overloading-to-create-a-complex-number-class.md).
+- [Implement and call a custom extension method](../programming-guide/classes-and-structs/how-to-implement-and-call-a-custom-extension-method.md).
+- Even C# programmers may want to [use the `My` namespace from VB](../programming-guide/namespaces/how-to-use-the-my-namespace.md).
+- [Create a new method for an `enum` type using extension methods](../programming-guide/classes-and-structs/how-to-create-a-new-method-for-an-enumeration.md).
+
+### Class and struct members
+
+You create classes and structs to implement your program. These techniques are common tasks when writing classes or structs.
```

Comment:
techniques and tasks are different things. Maybe "These techniques are commonly used when writing..."